Cargo.toml

```diff
@@ -139,6 +140,7 @@ metrics-exporter-prometheus = { version = "0.15", default-features = false, feat
     "async-runtime",
 ] }
 moka = "0.12.5"
+object_store = { version = "0.11.1", features = ["aws"] }
```

Comment:
Since we are not officially supporting anything other than S3, it was easier to just not compile the other providers. The file provider is always enabled, it seems.